src/main/resources/templates/goals/index.html

```diff
@@ -28,16 +28,19 @@
                 <div class="col-4">
                     <a class="btn btn-primary float-right" href="/create">Create a Goal</a>
                 </div>
-                <hr width="60%" align="left">
+                <hr width="100%" align="left">
             </div>
             <div class="row">
-                <div class="col-4">
+                <div class="col-3">
```

Comment:
This commit appears to be outdated since the col size is too small.

Reply:
Updated